handsontable/src/translations/indexMapper.js

```diff
@@ -656,22 +656,26 @@ export class IndexMapper {
    * @private
    * @param {number} firstInsertedVisualIndex First inserted visual index.
    * @param {number} amountOfIndexes Amount of inserted indexes.
-   */
-  insertIndexes(firstInsertedVisualIndex, amountOfIndexes) {
-    const nthVisibleIndex = this.getNotTrimmedIndexes()[firstInsertedVisualIndex];
-    const firstInsertedPhysicalIndex = isDefined(nthVisibleIndex) ? nthVisibleIndex : this.getNumberOfIndexes();
-    const insertionIndex = this.getIndexesSequence().includes(nthVisibleIndex) ?
-      this.getIndexesSequence().indexOf(nthVisibleIndex) : this.getNumberOfIndexes();
-    const insertedIndexes = arrayMap(new Array(amountOfIndexes).fill(firstInsertedPhysicalIndex),
+   * @param {number} firstInsertedPhysicalIndex First inserted physical index.
+   */
+  insertIndexes(firstInsertedVisualIndex, amountOfIndexes, firstInsertedPhysicalIndex) {
+    const maybeNotTrimmedPhysicalIndex = this.getNotTrimmedIndexes()[firstInsertedVisualIndex];
```

Comment:
I'm not a fan of using the "maybe" prefix for variables that might be `undefined`. It could lead to an unnecessary pattern where all such variables get the prefix (in the worst case), making the code more challenging to read or manage.

Reply:
I reverted this change to the original name